datafusion/physical-expr/Cargo.toml

```diff
@@ -47,6 +47,7 @@ chrono = { workspace = true }
 datafusion-common = { workspace = true, default-features = true }
 datafusion-expr = { workspace = true }
 datafusion-expr-common = { workspace = true }
+datafusion-functions = { workspace = true }
```

Comment:
When you open this PR to the upstream, we should ask if adding this dependency is a problem. Can you take a note reminding that?